dev/ci/ci-user-overlay.sh

```diff
@@ -0,0 +1,22 @@
+#!/usr/bin/env bash
+
+# Add user overlays here. You can use some logic to detect if you are
+# in your travis branch and conditionally enable the overlay.
```

Comment:
Anyways, regardless of whether the modification are kept in Coq or not, it seems a tad cleaner to me to do them in a separate file.